packages/gatsby-recipes/src/providers/gatsby/plugin.js

```diff
@@ -99,8 +99,8 @@ const getNameForPlugin = node => {
   return null
 }
 
-const getDescriptionForPlugin = async name => {
-  const pkg = await readPackageJSON({}, name)
+const getDescriptionForPlugin = async (root, name) => {
+  const pkg = await readPackageJSON({ root }, name)
```

Comment:
I might be reading it wrong but https://github.com/gatsbyjs/gatsby/pull/26699/files#diff-e78f1094a9c69cf340ad7054949c39dcR15 seems to be a string and not an object or do I miss something?
```suggestion
  const pkg = await readPackageJSON(root, name)
```

Reply:
yeah good catch